app/src/main/java/nl/tudelft/b_b_w/controller/KeyReader.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.b_b_w.controller;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Class to read the private key from the file
+ */
+public class KeyReader {
```

Comment:
Is this class really necessary? All it does is read a string from a file; why not do this in *initKey* method?